packages/flutter_tools/lib/src/test/flutter_web_goldens.dart

```diff
@@ -100,7 +100,11 @@ class TestGoldenComparator {
 
   Future<String?> compareGoldens(Uri testUri, Uint8List bytes, Uri goldenKey, bool? updateGoldens) async {
     final File imageFile = await (await tempDir.createTemp('image')).childFile('image').writeAsBytes(bytes);
-    final TestGoldenComparatorProcess process = await (_processForTestFile(testUri) as FutureOr<TestGoldenComparatorProcess>);
+    final TestGoldenComparatorProcess? process = await _processForTestFile(testUri);
+    if (process == null) {
+      return 'unknown error';
```

Comment:
never seen this code before, but I wonder should we leave a breadcrumb here that we hit this branch rather than the later unknown error at 113?

Reply:
The error is different now. not sure how useful it is, but at least we can trace it back here

Reply:
I don't always investigate failing golden tests, but when I do, it's a traumatic experience.